### ISSUE TITLE
Unknown system error -8

### ISSUE THREAD

**Issue by ad-si** [purescript/spago#1225]
In GitHub actions, [Transity](https://github.com/ad-si/Transity) builds fine on Ubuntu and macOS.
However, on my local Mac mini, I get following error:

```txt
$ npx spago test
file:///Users/adrian/Dropbox/Projects/Transity/node_modules/spago/bin/bundle.js:61138
                  throw util.fromLeft(step3);
                  ^

Error: spawn Unknown system error -8
    at ChildProcess.spawn (node:internal/child_process:421:11)
    at spawn (node:child_process:761:9)
    at file:///Users/adrian/Dropbox/Projects/Transity/node_modules/spago/bin/bundle.js:96987:18
    at runSync (file:///Users/adrian/Dropbox/Projects/Transity/node_modules/spago/bin/bundle.js:60795:21)
    at run16 (file:///Users/adrian/Dropbox/Projects/Transity/node_modules/spago/bin/bundle.js:60988:25)
    at file:///Users/adrian/Dropbox/Projects/Transity/node_modules/spago/bin/bundle.js:61004:23
    at drain (file:///Users/adrian/Dropbox/Projects/Transity/node_modules/spago/bin/bundle.js:60822:9)
    at Object.enqueue (file:///Users/adrian/Dropbox/Projects/Transity/node_modules/spago/bin/bundle.js:60840:11)
    at file:///Users/adrian/Dropbox/Projects/Transity/node_modules/spago/bin/bundle.js:60998:31
    at __do4 (file:///Users/adrian/Dropbox/Projects/Transity/node_modules/spago/bin/bundle.js:97818:18) {
  errno: -8,
  code: 'Unknown system error -8',
  syscall: 'spawn'
}

Node.js v20.9.0
```

<details><summary>Details of Mac mini</summary>
<p>
OS: macOS Sonoma 14.5 x86_64<br>
Host: Mac mini (2018)<br>
Kernel: Darwin 23.5.0<br>
Shell: fish 3.7.1<br>
Terminal: iTerm 3.5.0beta8<br>
CPU: Intel(R) Core(TM) i7-8700B (12) @ 3.20 GHz<br>
GPU: Intel UHD Graphics 630 [Integrated]<br>
Memory: 18.88 GiB / 32.00 GiB (59%)<br>
Swap: Disabled<br>
Disk (/): 904.56 GiB / 931.55 GiB (97%) - apfs [Read-only]<br>
Locale: en_US.UTF-8<br>
</p>
</details> 

Any idea what could cause this?

**Comment by ad-si** [comment]
This happens with Node.js v22 but not with v20

**Comment by f-f** [comment]
@ad-si I looked at this but couldn't find anything conclusive - however in #1229 there was another case of "works with Node 20 but not 22 on mac", and the fix for that was simply to upgrade all our npm dependencies. I just released 0.93.32 with the fix, could you try that one?

**Comment by ad-si** [comment]
Yes, that fixed it! (https://github.com/ad-si/Transity/commit/8d6c41db98b422f52c95479b37f77b113b345e9e)
Thanks a lot!
